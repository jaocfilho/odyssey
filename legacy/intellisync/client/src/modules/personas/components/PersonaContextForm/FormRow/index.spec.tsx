import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';

import { FormRow } from '.';

describe('FormRow', () => {
  it('should render without crashing', () => {
    const { container } = render(<FormRow>children</FormRow>);
    expect(container).toMatchSnapshot();
  });
});
