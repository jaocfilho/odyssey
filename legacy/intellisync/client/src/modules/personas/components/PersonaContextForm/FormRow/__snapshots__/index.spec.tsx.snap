// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FormRow > should render without crashing 1`] = `
<div>
  <li
    class="relative flex gap-x-4"
  >
    <div
      class="-bottom-6 absolute left-0 top-0 flex w-6 justify-center"
    >
      <div
        class="w-px dark:bg-grayScheme-50/10"
      />
    </div>
    children
  </li>
</div>
`;
